src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1301,6 +1301,11 @@ private async Task SetBreakpoint(SessionId sessionId, DebugStore store, Breakpoi
             {
                 SourceLocation loc = sourceId.First();
                 req.Method = loc.CliLocation.Method;
+                if (req.Method.IsHiddenFromDebugger)
+                {
```

Comment:
You can remove the `{` here

Reply:
Resolved with [commit](https://github.com/dotnet/runtime/pull/61143/commits/3deab4b59924da80b828ac016169cc967d2c717e).